jdi-light-applitools-tests/src/test/java/com/jdi/states/State.java

```diff
@@ -0,0 +1,28 @@
+package com.jdi.states;
+
+/**
+ * Created by Roman Iovlev on 10.11.2018
+ * Email: roman.iovlev.jdi@gmail.com; Skype: roman.iovlev
+ */
+
+import jdisite.entities.DefaultData;
+
+import static jdisite.pages.HomePage.*;
+import static jdisite.pages.JDISite.*;
+
+public class State {
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [The utility class name 'State' doesn't match '[A-Z][a-zA-Z0-9]+(Utils?|Helper|Constants)'](https://app.codacy.com/manual/42_/jdi-light/pullRequest?prid=5511453)